.github/workflows/batch_api_ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: Run batch-api tests
+
+on:
+  release:
+    types: [ published ]
+  pull_request:
+    paths:
+      - 'flexeval/core/language_model/*.py'
+      - 'tests/core/language_model/*.py'
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  run-batch-api-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # 3.11.9, which is picked up by GithubAction by default, doesn't work because of argparse or something...
+        python-version: ["3.9", "3.10", "3.11.6"]
+        test-marker: ["", "Batch API"]
```

Comment:
It seems not used any more. Remove this line?

Reply:
Oops, nice catch! Will do that

Reply:
I just removed an entire matrix. To me, it seemed that there is no strong reason to run this heavy test for every Python version. How do you think?

Reply:
Yes, I think it is reasonable to run the test for a single python version.
Syntax/behavior discrepancies across Python versions should be rare.